packages/nx/src/project-graph/plugins/utils.ts

```diff
@@ -1,62 +1,9 @@
-import { dirname } from 'node:path';
-
-import { toProjectName } from '../../config/to-project-name';
-import { combineGlobPatterns } from '../../utils/globs';
-
-import type { NxPluginV1 } from '../../utils/nx-plugin.deprecated';
-import type { LoadedNxPlugin, NormalizedPlugin } from './internal-api';
 import {
```

Comment:
Are there changes to the V2 public APIs? I see that `CreateNodesV2` is slated for removal in v21, and users should be using `CreateNodes` instead. Is that part of v20 changes?